### DISCUSSION TITLE
can I build docker cli on my own?

### DISCUSSION THREAD

**Discussion by heyyuyu** [mockoon/mockoon#1361] [Help]
Since https://hub.docker.com/r/mockoon/cli is not available in my country,I can't get the latest version mockoon_cli,how can I build docker cli on my own,is there any guidance?

**Comment by 255kb** [comment]
You can either use the [`dockerize` command](https://github.com/mockoon/mockoon/tree/main/packages/cli#using-the-dockerize-command) that generates a Dockerfile, and then run `docker build` and `docker run`, or you can build from [the Dockerfile we use to build the official image](https://github.com/mockoon/mockoon/blob/main/packages/cli/docker/Dockerfile).
It uses NPM to pull the package. If you don't have access to NPM you should be able to clone and build the CLI using the [contributing guidelines](https://github.com/mockoon/mockoon/blob/main/CONTRIBUTING.md#build-and-run-the-applications-locally-during-development).
